integration.js

```diff
@@ -300,6 +301,34 @@ async function onMessage(payload, options, cb) {
         });
       }
       break;
+    case 'ADD_INTEGRATION_DATA_AS_NOTE':
+      Logger.info('Adding integration data as note', payload);
+      Logger.info('Options:', options.enableAddingNotes);
+      try {
+        let notes;
+        if (!options.enableAddingNotes) {
+          return cb(null, {
```

Comment:
Will comment in the component file but you are sending back an error here and not catching it in the component.  As a result, the component  file never shows this error message if the notes are not enabled.